.github/workflows/repro-check.yml

```diff
@@ -0,0 +1,60 @@
+name: Repro check
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      github_sha:
+        description: Commit on which the repro check should be run
+        required: true
+        type: string
+  # Used as a reusable workflow within release-testing workflow
+  workflow_call:
+    inputs:
+      github_sha:
+        description: Commit on which the repro check should be run
+        required: true
+        type: string
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  repro-check:
+    name: Run repro-check for ${{ inputs.github_sha }}
+    runs-on:
+      labels: dind-small
```

Comment:
out of curiosity have you tried running it on GitHub or namespace runners?

Reply:
Not really, because the CI lasts ATM for about 2h 15mins and its just going to smash the bill for no reason. Do you see any benefits it could bring?

Reply:
our runners (and the images they use) might include tweaks that could create false positives where the test would pass whereas external devs might not be able to reproduce. 2h every week is not too bad I'd say, but up to you!

Reply:
We use our image anyway (the dev container is used from within the script). And as for the starter I put the exact same os that would be if I did `ubuntu-latest` we could discuss further. 

It would actually result in + 1.2~1.4h difference because it runs in parallel with qualification which also takes quite some time to complete. The additional overhead is on a daily bases and it would slow down urgent hotfixes which is why I still left this in Draft.